lib/order.rb

```diff
@@ -0,0 +1,62 @@
+require_relative 'menu'
+
+class Order
+
+  attr_reader :order, :menu_list
+
+  def initialize
+    @order = Hash.new(0)
+  end
+
+  def check_menu
+    puts menu_list
+  end
+
+  def add_order(name, quantity)
+      @order[name] = quantity
+      order.each {|key, value| puts "you have added #{key} times #{value} to your order" }
+      @order
+  end
+
+  def order_sum
+      raise 'The basket is empty' if order.empty?
```

Comment:
Style/SignalException: Use `fail` instead of `raise` to signal exceptions. (https://github.com/bbatsov/ruby-style-guide#fail-method)